lib/core/mergeConfig.js

```diff
@@ -55,12 +55,27 @@ module.exports = function mergeConfig(config1, config2) {
     .concat(mergeDeepPropertiesKeys)
     .concat(defaultToConfig2Keys);
 
+  var defaultKeys = Object
```

Comment:
concat `defaultKeys` and `otherKeys`, then handle them with `utils.Foreach` will reduce duplicate code, what do you think?

Reply:
Done.